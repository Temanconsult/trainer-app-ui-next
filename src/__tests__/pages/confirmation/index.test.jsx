import React from 'react'
import { render } from '@testing-library/react'
import Confirmation from '../../../pages/confirmation'

describe('Pages:Confirmation', () => {
  it('renders the confirmation page', () => {
    const { asFragment } = render(<Confirmation />)
    expect(asFragment()).toMatchSnapshot()
  })
})
