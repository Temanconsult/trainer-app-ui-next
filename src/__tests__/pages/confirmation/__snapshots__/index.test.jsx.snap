// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pages:Confirmation renders the confirmation page 1`] = `
<DocumentFragment>
  <p>
    Boilerplate content for confirmation
  </p>
  <p>
    Coming Soon
  </p>
</DocumentFragment>
`;
